tests/messages.cpp

```diff
@@ -162,6 +162,49 @@ TEST(RoomEvents, ConfettiMessage)
     EXPECT_EQ(data.dump(), json(event).dump());
 }
 
+TEST(RoomEvents, UnknownMessage)
```

Comment:
I'd add another test for the 'Invalid' case that covers the changes you made to `lib/struct/events.cpp`  .

Reply:
The Invalid case should be covered by the existing test at https://github.com/Nheko-Reborn/mtxclient/blob/ducktyping2/tests/responses.cpp#L998